lk2nd/device/dts/msm8916/msm8939-mtp.dts

```diff
@@ -34,6 +34,17 @@
 		};
 	};
 
+	sony-tulip {
+		model = "Sony Xperia M4 Aqua";
+		compatible = "sony,kanuti-tulip";
+		/*
+		 * Similar to xperia-leo, this match is probably too broad, but so far
+		 * this is the only Sony device in the msm8916 family.
```

Comment:
Here's the cmdline this device got from previous bootloader:

```
[50] Command line from previous bootloader: lk2nd androidboot.emmc=true androidboot.bootloader=s1 oemandroidboot.s1boot=1291-8005_S1_Boot_MSM8939_LA2.0_207 androidboot.serialno=YT9113Y4JU ta_info=1,16,256 startup=0x00008004 warmboot=0x00000000 oemandroidboot.imei=<snip> oemandroidboot.phoneid=<snip> oemandroidboot.security=0 oemandroidboot.babeee48=00000200 oemandroidboot.securityflags=0x00000003 display_status=on androidboot.baseband=msm mdss_mdp.panel=0:dsi:0:
```

Open to suggestions on a better match here.

The truncated panel string is weird, but maybe that's because lk2nd was booted manually via `fastboot boot`?